app/routes.py

```diff
@@ -1,2 +1,88 @@
-from flask import Blueprint
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify,abort, make_response, request
+ 
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets") 
 
+@planets_bp.route("", methods=["GET"])
+def get_all_planets():
+    name_query = request.args.get("name")
+    if name_query:
+        planets = Planet.query.filter_by(name = name_query)
+    else:
+        planets = Planet.query.all()
+    planets_response = []  
+    for planet in planets:                                                 
+        planets_response.append({                                  
+        "id": planet.id,                                   
+        "name": planet.name,
+      })                                         
```

Comment:
if I'm not querying by name then I want to get all the information back from the planet. So I would want id, name, description, and moons.